ui/components/app/set-approval-for-all-warning/set-approval-for-all-warning.stories.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import SetApproveForAllWarning from '.';
+
+export default {
+  title: 'Components/App/SetApproveForAllWarning',
+  id: __filename,
+};
+
+export const DefaultStory = (args) => {
+  return (
+    <SetApproveForAllWarning
+      collectionName={args.collectionName}
+      name={args.name}
+      senderAddress={args.senderAddress}
+      total={args.total}
+    />
+  );
+};
+
+DefaultStory.storyName = 'Default';
+DefaultStory.args = {
+  collectionName: 'Test collection',
+  name: 'Account 1',
+  senderAddress: '0xee014609ef9e09776ac5fe00bdbfef57bcdefebb',
+  total: '6',
+};
```

Comment:
Nice! You can add the argTypes and default args to the storybook options as well as add the action controls

```suggestion
  argTypes: {
    collectionName: {
      control: 'text',
    },
    name: {
      control: 'text',
    },
    senderAddress: {
      control: 'text',
    },
    total: {
      control: 'text',
    },
    onSubmit: {
      action: 'onSubmit',
    },
    onCancel: {
      action: 'onCancel',
    },
  },
  args: {
    collectionName: 'Test collection',
    name: 'Account 1',
    senderAddress: '0xee014609ef9e09776ac5fe00bdbfef57bcdefebb',
    total: '6',
  },
};

export const DefaultStory = (args) => <SetApproveForAllWarning {...args} />;

DefaultStory.storyName = 'Default';
```